image-loader/app/controllers/ImageLoaderController.scala

```diff
@@ -72,9 +74,8 @@ class ImageLoaderController(auth: Authentication,
           DateTimeUtils.fromValueOrNow(uploadTime),
           filename.flatMap(_.trim.nonEmptyOpt),
           context.requestId)
-        result <- uploader.storeFile(uploadRequest)
+        result <- if (config.uploadToQuarantineEnabled && !config.quarantineBucket.isEmpty) quarantineUploader.get.quarantineFile(uploadRequest) else uploader.storeFile(uploadRequest)
```

Comment:
You've already checked the config when figuring out whether to create the QuarantineUploader. So this can simply be `if (quarantineUploader.nonEmpty) quarantineUploader.get.quarantineFile(uploadRequest) else uploader.storeFile(uploadRequest)`. Or, more idiomatically `quarantineUploader.map(_.quarantineFile(uploadRequest)).getOrElse(uploader.storeFile(uploadRequest)`

Might be nicer to put it into a small function at the top so as to express the intent more clearly.